.circleci/config.yml

```diff
@@ -239,38 +207,47 @@ jobs:
                   keys:
                       - repo-{{ .Environment.CIRCLE_SHA1 }}
             - run: yarn wsrun test:circleci @0x/contracts-test-utils
-            # TODO(dorothy-zbornak): Re-enable after updating this package for 3.0.
-            # - run: yarn wsrun test:circleci-3.0 @0x/abi-gen
+            - run: yarn wsrun test:circleci @0x/abi-gen
+            - run: yarn wsrun test:circleci @0x/asset-buyer
+            - run: yarn wsrun test:circleci @0x/contract-artifacts
             - run: yarn wsrun test:circleci @0x/assert
             - run: yarn wsrun test:circleci @0x/base-contract
-            # TODO(dorothy-zbornak): Re-enable after updating this package for 3.0.
```

Comment:
We haven't been building ts packages in the `3.0` branch. The plan is to refactor packages piecemeal in subsequent PRs. For now we're just trying to reconcile `development` changes with `3.0`, and then continue working on `3.0`

Reply:
👍 I see. I think it would be less confusing if (rather than adding separate steps with a `3.0` suffix) we simply commented or changed the existing steps as needed in order to get the tests working on this branch. But if you are happy with the current state of __.circleci/config.yml__ I'm okay with it. I care a lot more about the end state of the config file after we get all the tests passing.